server/controllers/finance/debtors/index.js

```diff
@@ -134,8 +134,10 @@ function invoices(req, res, next) {
   const options = req.query;
 
   // get the debtor invoice uuids from the invoice table
-  let sql =
-    'SELECT uuid FROM invoice WHERE debtor_uuid = ?;';
+  let sql =`
+    SELECT invoice.uuid 
+    FROM invoice 
+    WHERE debtor_uuid = ? AND invoice.uuid NOT IN (SELECT voucher.reference_uuid FROM voucher WHERE voucher.type_id = 10);`;
```

Comment:
The voucher type_id should be defined as a variable here. 

At some point we should have a set of system constants on the server that ensure the same numbers are used everywhere and can be updated easily. For now a variable separating the value and the concept is okay. 

For example:
```js 
const REVERSAL_VOUCHER_TYPE = 10;
let sql = `...
  ... WHERE voucher.type_id = ${REVERSAL_VOUCHER_TYPE}
```